### PR TITLE
Update to use rtm.connect

### PR DESCRIPTION
**Pull request by jacobtruman**

### CODE REVIEW

File: oncall_slackbot/slackclient.py [hunk 1]
```diff
@@ -27,6 +31,11 @@ def __init__(self, token, timeout=None, bot_icon=None, bot_emoji=None, connect=T
         if connect:
             self.rtm_connect()
 
+    def rtm_connect(self):
+        reply = self.webapi.rtm.connect().body
+        time.sleep(1)
```
Comment (bluesliverx) [inline oncall_slackbot/slackclient.py:36]: Why a sleep?

Comment (jacobtruman) [reply oncall_slackbot/slackclient.py:36]: copied from the method I am overriding:
https://github.com/scrapinghub/slackbot/blob/develop/slackbot/slackclient.py#L45-L48

Comment (bluesliverx) [reply oncall_slackbot/slackclient.py:36]: Actual commit, looks like it was done to "increase delays". That makes sense...

https://github.com/scrapinghub/slackbot/commit/4c8a2423e8b936a734369ce5d8ac269e0386ee67

Oh well, LGTM!

Comment (jacobtruman) [reply oncall_slackbot/slackclient.py:36]: well, I guess it does increase delays, though I am not sure why that was wanted...

**Review by bluesliverx — APPROVED** [review]
lgtm-code-review

**Pull request merged**